metaci/build/models.py

```diff
@@ -686,6 +682,15 @@ def run_flow(self, project_config, org_config):
         # Run the flow
         return self.flow_instance.run(org_config)
 
+    def set_release_dates(self, options):
```

Comment:
Let's call this `_get_flow_options` instead of `set_release_dates`. It's not setting anything. It's currently only adding the dates, but it would be the natural place to add other flow options. The initial underscore is an indicator that it's not meant to be called from elsewhere in the codebase.